### ISSUE TITLE
Dropdown in Carousel

### ISSUE THREAD

**Issue by zephyr-dev** [twbs/bootstrap#17413]
Problem: Putting a button dropdown inside of a carousel item results in the dropdown menu items getting cut off.

Example: http://codepen.io/anon/pen/epYgVa

I know this is because the `.carousel` class has `overflow: hidden`. I can't think of a reasonable fix for this. Any ideas?

**Comment by twbs-lmvtfy** [comment]
Hi @zephyr-dev!

You appear to have posted a live example (http://codepen.io/anon/pen/epYgVa.html), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 38, column 5 thru column 70: Attribute ```` aria_hidden ```` not allowed on element ```` span ```` at this point.
* line 41, column 5 thru column 71: Attribute ```` aria_hidden ```` not allowed on element ```` span ```` at this point.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by zephyr-dev** [comment]
The markup has been fixed.

**Comment by cvrebert** [comment]
Forcing revalidation: http://codepen.io/anon/pen/epYgVa

**Comment by twbs-lmvtfy** [comment]
Hi @cvrebert!

You appear to have posted a live example (http://codepen.io/anon/pen/epYgVa.html), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 11, column 7: [E003](https://github.com/twbs/bootlint/wiki/E003): Found one or more `.row`s that were not children of a grid column or descendants of a `.container` or `.container-fluid`

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by zephyr-dev** [comment]
Validating again... http://codepen.io/anon/pen/epYgVa

**Comment by Teaflavored** [comment]
This seems like the intended behavior. carousel-inner has overflow hidden applied on it to hide extra items as they are being slided across. I would recommend attaching event on the btn and adding dropdown options on say body element and position it the correct place

**Comment by Teaflavored** [comment]
Another solution is to remove the overflow hidden on the carousel inner and add another wrapper div with overflow hidden, the key is the make the wrapper div taller to contain whatever dropdown data you'll have in there. Also in the first place I wouldn't say putting a dropdown in a carousel is a good idea.

**Comment by shanksdev** [comment]
@zephyr-dev, in my opinion, if you want that dropdown in your carousel at that position only, or any place at the bottom of the carousel, instead of dropping down the menu you can simply display the menu of options in upward direction, using class '.dropup' instead of '.dropdown'. The reverse thing can be followed if you want to put your dropdown somewhere on the top. But seriously, putting dropdown inside carousel doesn't seem like a good idea.

**Comment by mdo** [comment]
Sorry, but we'll be closing as a won't fix. No way around this unless you want to override the defaults (which need to be there).
